core/contracts/financial-templates/implementation/SyntheticToken.sol

```diff
@@ -4,76 +4,73 @@ import "../../common/implementation/ExpandedERC20.sol";
 
 
 /**
- * @notice A burnable and mintable ERC20. The contract deployer will initially
- * be the only minter and burner as well as the owner who is capable of adding new roles.
- * The contract deployer will also be the only initial minter of the contract.
+ * @title Burnable and mintable ERC20.
+ * @dev The contract deployer will initially be the only minter and burner as well as the owner  who
+ * is capable of adding new roles. The contract deployer will also be the only initial minter of the contract.
  */
+
 contract SyntheticToken is ExpandedERC20, ERC20Detailed {
+    /**
```

Comment:
Just checking, have you always made comments for constructors?

Reply:
yup, always. Can strip them out of all the contracts if we want but to be consistent they have always been there